Cargo.toml

```diff
@@ -8,3 +8,10 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.3.8", features = ["derive"] }
 lazy-regex = "2.5.0"
+rustyline = "=5.0.2"
```

Comment:
Why is this a specific version?

Reply:
the command line provided to install this library in the crates website was 'rustyline = "=5.0.2"', I just use it exactly how it was provided. Any way, I found another library that I think that may be better, but idk, I will look more about it.